extensions/amp-consent/0.1/consent-config.js

```diff
@@ -150,6 +150,31 @@ export class ConsentConfig {
       }
     }
 
+    userAssert(
+      config['checkConsentHref'] ||
+        config['promptIfUnknownForGeoGroup'] ||
+        config['consentRequired'],
+      'neither checkConsentHref, promptIfUnknownForGeoGroup, nor consentRequired is defined'
+    );
+
+    // consentRequired must not be null
+    if (config['checkConsentHref'] && !config['consentRequired']) {
+      config['consentRequired'] = 'remote';
+    } else if (!config['consentRequired']) {
+      // Should we be conservative and make it true?
+      config['consentRequired'] = false;
+    }
+
+    const group = config['promptIfUnknownForGeoGroup'];
```

Comment:
move this `promptIfUnknownForGeoGroup` handling before `consentRequired`

Reply:
What case are you concerned about?

Reply:
Actually, I see it now.